dts/riscv/sophgo/cv180x.dtsi

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2023-2024 Chen Xingyu <hi@xingrz.me>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+
+#include <sophgo/cv18xx-c906-common.dtsi>
+
+/ {
+	soc {
+		compatible = "sophgo,cv180x";
+
+		mbox: mailbox@1900000 {
+			compatible = "sophgo,cvi-mailbox";
+			reg = <0x1900000 DT_SIZE_K(4)>;
+			interrupt-parent = <&plic0>;
```

Comment:
why isn't interrupt parent defined ad the soc node? this way you avoid definit it on every child node